src/cascadia/TerminalSettingsModel/ActionMap.cpp

```diff
@@ -857,6 +857,18 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         AddAction(*cmd);
     }
 
+    void ActionMap::AddSendInputAction(winrt::hstring name, winrt::hstring input)

```

Comment:
I tried to add this in TerminalPage::_HandleSaveTask,  the build errors from calling winrt::make<SendInputArgs>(input), defeated me.